README.md

```diff
@@ -1,5 +1,7 @@
 # Getting Started with Create React App
 
+## [LIVE LINK](https://vast-meadow-84545.herokuapp.com)
```

Comment:
General Requirement Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
If you check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

kindly add the following which is the minimum checklist for this README FILE for now;
- screenshot from your app. (this could be added in milestone 5)
- contact address, that is; Author
- setup 

Set up example:
```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```